### ISSUE TITLE
A Folding search

### ISSUE THREAD

**Issue by Th4nat0s** [CIRCL/lookyloo#24]
Hello,

It would be nice to have a "search" which will find and unfold only the relevant path to the result of the search.

**Comment by Rafiot** [comment]
Yeah, that one is super tricky, the data are stored all over the place and displaying just one path is hard... I'll see what I can do.

**Comment by Rafiot** [comment]
Try that: https://scout.readthedocs.io/en/latest/index.html
